client/components/list-settings.vue

```diff
@@ -115,6 +120,11 @@ export default {
     },
     beforeMount: function() {
         this.updateOptionalFieldValues();
+    },
+    mounted: function() {
+        bus.$on("optionalFieldChanged", () => {

```

Comment:
In general I'd like to reduce the reliance on the event bus, but it looks like this page isn't built too well right now 😬 so this works!